src/blob/index/core.rs

```diff
@@ -59,14 +59,34 @@ struct MemoryAttrs<K> {
     marker: PhantomData<K>,
 }
 
+const BTREE_B_FACTOR: usize = 6;
+const BTREE_VALUES_LEN: usize = BTREE_B_FACTOR * 2 - 1;
+const BTREE_EDGES_LEN: usize = BTREE_B_FACTOR * 2;
+
 impl<K> MemoryAttrs<K>
 where
     for<'a> K: Key<'a>,
 {
-    const KEY_SIZE: usize = K::LEN as usize;
-    const BPTREE_ENTRY_SIZE: usize =
-        size_of::<Vec<u8>>() + MemoryAttrs::<K>::KEY_SIZE + size_of::<Vec<RecordHeader>>();
-    const RECORD_HEADER_SIZE: usize = size_of::<RecordHeader>() + MemoryAttrs::<K>::KEY_SIZE;
+    const BTREE_ENTRY_SIZE: usize =
+        size_of::<Vec<u8>>() + K::MEM_SIZE + size_of::<Vec<RecordHeader>>();
+    const RECORD_HEADER_SIZE: usize = size_of::<RecordHeader>() + K::MEM_SIZE;
```

Comment:
Here, the previous implementation was correct. It should be `size_of::<RecordHeader>() + MemoryAttrs::<K>::KEY_SIZE`, because `RecordHeader` stores key inside a vector